k8s/nginx-ws/container/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nginx:1.13.1
+LABEL maintainer "dev@bigchaindb.com"
+WORKDIR /
+RUN apt-get update \
+    && apt-get -y upgrade \
+    && apt-get autoremove \
+    && apt-get clean
+COPY nginx_ws.conf.template /etc/nginx/nginx.conf
+COPY nginx_ws_entrypoint.bash /
+EXPOSE 81 444 9985
+ENTRYPOINT ["/nginx_ws_entrypoint.bash"]
```

Comment:
@libscott It's opinionated, and differs from person to person, and team to team. A filename is just a name under *nix.
Personally, I use it so that a user/engineer can easily find at a glance (using `ls` or something) that it's a bash script, rather than open the file to read the first line. Truly portable scripts can use `.sh`, but I would still want to show what a file contains from a name.

Also, that is the convention followed in all the other modules under `k8s/` now, and I would prefer to stick with it. :)